include/seqan3/io/alignment_file/input.hpp

```diff
@@ -490,8 +459,29 @@ class alignment_file_input
                                   bitscore_type,
                                   header_type *>;
 
-    /*!\brief The subset of seqan3::field tags that are valid for this file; order corresponds to the types in
-     * \ref field_types.
+    /*!\brief The subset of seqan3::field tags valid for this file; order corresponds to the types in \ref field_types.
+     *
+     * The Alignment file abstraction supports reading 15 different fields:
+     *
+     *   1. seqan3::field::seq
+     *   2. seqan3::field::id
+     *   3. seqan3::field::offset
+     *   4. seqan3::field::ref_seq
+     *   5. seqan3::field::ref_id
+     *   6. seqan3::field::ref_offset
+     *   7. seqan3::field::alignment
+     *   8. seqan3::field::cigar
+     *   9. seqan3::field::mapq
+     *   10. seqan3::field::qual
+     *   11. seqan3::field::flag
+     *   12. seqan3::field::mate
+     *   13. seqan3::field::tags
+     *   14. seqan3::field::evalue
+     *   15. seqan3::field::bit_score
+     *
+     * There exists one more field for alignment files, the seqan3::field::header_ptr, but this field is mostly used
```

Comment:
why does this need to be a field in the record?

Reply:
Because if I do `fin | fout` (range interface) I need to pass the header information somehow